addons/src/createAutoCorrectedDatePipe.js

```diff
@@ -1,9 +1,9 @@
-export default function createAutoCorrectedDatePipe(dateFormat = 'mm dd yyyy') {
+export default function createAutoCorrectedDatePipe(dateFormat = 'mm dd yy yyyy') {
```

Comment:
@mathieu-neron why have you added the `yy` to the default date format.. there is no need to change the default.